Tests/ASScrollNodeTests.m

```diff
@@ -16,48 +16,49 @@
 #import "ASXCTExtensions.h"
 
 @interface ASScrollNodeTests : XCTestCase
+
+@property (nonatomic) ASScrollNode *scrollNode;
+@property (nonatomic) ASDisplayNode *subnode;
+
 @end
 
-@implementation ASScrollNodeTests {
-  ASScrollNode *scrollNode;
-  ASDisplayNode *subnode;
-}
+@implementation ASScrollNodeTests
 
 - (void)setUp
 {
-  subnode = [[ASDisplayNode alloc] init];;
-
-  scrollNode = [[ASScrollNode alloc] init];
-  scrollNode.scrollableDirections = ASScrollDirectionVerticalDirections;
-  scrollNode.automaticallyManagesContentSize = YES;
-  scrollNode.automaticallyManagesSubnodes = YES;
-  __weak __typeof(self) weakSelf = self;
-  scrollNode.layoutSpecBlock = ^ASLayoutSpec * _Nonnull(__kindof ASDisplayNode * _Nonnull node, ASSizeRange constrainedSize) {
-    __strong __typeof(weakSelf) strongSelf = weakSelf;
-    return [[ASWrapperLayoutSpec alloc] initWithLayoutElement:strongSelf.subnode];
+  ASDisplayNode *subnode = [[ASDisplayNode alloc] init];
+  self.subnode = subnode;
+
+  self.scrollNode = [[ASScrollNode alloc] init];
+  self.scrollNode.scrollableDirections = ASScrollDirectionVerticalDirections;
+  self.scrollNode.automaticallyManagesContentSize = YES;
+  self.scrollNode.automaticallyManagesSubnodes = YES;
+  self.scrollNode.layoutSpecBlock = ^ASLayoutSpec * _Nonnull(__kindof ASDisplayNode * _Nonnull node, ASSizeRange constrainedSize) {
+    return [[ASWrapperLayoutSpec alloc] initWithLayoutElement:subnode];
   };
+  [self.scrollNode view];
 }
 
 - (void)testSubnodeLayoutCalculatedWithUnconstrainedMaxSizeInScrollableDirection
 {
   CGSize parentSize = CGSizeMake(100, 100);
   ASSizeRange sizeRange = ASSizeRangeMake(parentSize);
 
-  [scrollNode layoutThatFits:sizeRange parentSize:parentSize];
+  [self.scrollNode layoutThatFits:sizeRange parentSize:parentSize];
 
   ASSizeRange subnodeSizeRange = sizeRange;
   subnodeSizeRange.max.height = CGFLOAT_MAX;
-  XCTAssertEqual(scrollNode.scrollableDirections, ASScrollDirectionVerticalDirections);
-  ASXCTAssertEqualSizeRanges(subnode.constrainedSizeForCalculatedLayout, subnodeSizeRange);
+  XCTAssertEqual(self.scrollNode.scrollableDirections, ASScrollDirectionVerticalDirections);
```

Comment:
Rather than self. everywhere, using the ivar like _scrollNode would be more readable (especially in property chains & methods that use it a lot).

Reply:
As a follow-up to this, I find that using non-underscored ivar names in test classes provides the best readability.

Since the object is destroyed after `-tearDown` is run, and since test classes usually have trivial accessors, direct ivar access is safe and convenient.